Makefile

```diff
@@ -208,11 +220,31 @@ container:
 ifneq ($(BUILDX_ENABLED), true)
 	$(error $(BUILDX_ERROR))
 endif
+	-docker buildx rm velero-builder || true
+	@docker buildx create --use --name=velero-builder
```

Comment:
This overrides my current buildx builder settings. ie. I already have a kubernetes docker builder I intended to use on a more powerful system.

Can we do it in currently available builders that our docker is already set to use?

I think this can be doc as a pre-step. It is also not compatible with podman for example.

```
~/git/velero build-hybrid-image 1m 44s
❯ podman buildx ls
Error: unrecognized command `podman buildx ls`
Try 'podman buildx --help' for more information

~/git/velero build-hybrid-image
❯ podman buildx rm
Error: unrecognized command `podman buildx rm`
Try 'podman buildx --help' for more information

~/git/velero build-hybrid-image
❯ podman buildx create
Error: unrecognized command `podman buildx create`
Try 'podman buildx --help' for more information
```

Reply:
```
BUILD_OUTPUT_TYPE=registry REGISTRY=xxx BUILD_OS=windows make container
```
other than that this did build.

Reply:
I have modified the code to accept specified buildx instance by `BUILDX_INSTANCE` input parameter.

Reply:
For the podman part, as far as my understanding, there is no conclusions/actions to support podman emulated docker purposefully. Looks like the current Makefile works, it may be a coincidence.
Anyway, with `BUILDX_INSTANCE` input parameter set, we don't add any new `docker buildx` call, hopefully this could ensure podman still work as is.
However, supporting podman is not something must-have with/without this PR. If podman is required after our evaluation, there should be separate issue and PR for the necessary changes based on this PR.

Reply:
For the doc part, there is a separate issue for doc chagnes related to Windows support #8429, so there will be a separate PR for the same which will include the changes mentioned here.